### DISCUSSION TITLE
Multiple Mac Security Warnings when opening files

### DISCUSSION THREAD

**Discussion by crhuber** [helix-editor/helix#9468] [Q&A]
I installed Helix on mac by downloading the github release, unzipping it, adding it to path, copying the runtime folders to ~/.helix and setting
`HELIX_RUNTIME=~/.helix/runtime`

<img width="258" alt="image" src="https://github.com/helix-editor/helix/assets/5857391/6990028b-5d33-4e86-a925-fc66f4dab4bb">

Now, whever I browse a directory using helix i get this popup for every file type. How can I remove these without using homebrew on mac

**Comment by klkl85** [comment]
In case anybody else is still running into this then I got the solution from [Stack Overflow Question # 73872164](https://stackoverflow.com/questions/73872164/allowing-many-unverified-apps-to-run-in-macos)

All the *.so library files are in the helix runtime grammars directory so the relevant, for me, command was;

`xattr -d com.apple.quarantine ~/.config/helix/runtime/grammars/*`

You should double check where your runtime is / consult the Helix documentation.
